lib/nadia.ex

```diff
@@ -330,6 +330,23 @@ defmodule Nadia do
   def get_file(file_id), do: request("getFile", file_id: file_id)
 
   @doc """
+  Use this method to get link for file for subsequent use.
+  This method is an extension of the `get_file` method.
+
+  *Attention: link will be valid for 1 hour.*
+
+  Args:
+  * `file_id` - File identifier to get link
```

Comment:
please add test code.
if you implement this method like https://github.com/zhyu/nadia/pull/16/files#r53564223, please have a look at [doctests](http://elixir-lang.org/getting-started/mix-otp/docs-tests-and-with.html#doctests)